### DISCUSSION TITLE
You have an error in your SQL syntax

### DISCUSSION THREAD

**Discussion by rodeoclassifieds** [geodesicsolutions-community/geocore-community#215] [General]
I can place an ad and see it as pending but when I go to approve the ad, nothing shows on the list.  When I look at view all orders for the users orders there is nothing on their list, but near the bottom of that page I see the following error:

error: SELECT
		o.id as order_id, o.status, o.buyer, o.created, o.admin,
		i.id AS invoice_id, u.id as user_id,
		u.username,
		o_r.val_string as gateway_type
		FROM `geodesic_order` AS o,`geodesic_invoice` AS i,
			`geodesic_order_registry` as o_r, `geodesic_logins` as u
		WHERE i.order = o.id AND o_r.order = o.id AND o_r.`index_key` = 'payment_type' AND u.id = o.buyer AND o.seller = 0 AND u.username=?
		GROUP BY o.id DESC
		ORDER BY order_id DESC
		LIMIT 0, 20

1

You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'DESC
		ORDER BY order_id DESC
		LIMIT 0, 20' at line 9

My host recently migrated me to a new server......suspecting something with that caused this as it was fine before.     They tried changing my php version back to 5.4 and I placed a new ad and still had the same issue with the new one.   Any advice?

**Comment by rodeoclassifieds** [comment]
Sorry I don't see how to delete this, I have also posted under "Issues"

**Comment by vicos59** [comment]
Duplicate of #216
